### PR TITLE
Create behaviour.csl

### PR DESCRIPTION
**Pull request by damnation333**
via https://forums.zotero.org/discussion/46757/style-request-behaviour
adapted for journals, books and chapters.

### CODE REVIEW

File: behaviour.csl [hunk 1]
```diff
@@ -0,0 +1,378 @@
...
+          <name and="symbol" initialize-with=". " delimiter=", "/>
+          <label form="short" prefix=", " text-case="title"/>
+        </names>
+      </if>
+    </choose>
+  </macro>
+  <macro name="author">
+    <names variable="author">
+      <name and="symbol" initialize-with="." name-as-sort-order="all"/>
+      <label form="short" strip-periods="true" prefix=", "/>
```
Comment (adam3smith) [inline behaviour.csl]: I find it unlikely that it's ed/eds here and ed./eds. above (and Ed./Eds. for secondary contributors) likely all going to be identical.

File: behaviour.csl [hunk 2]
```diff
@@ -0,0 +1,378 @@
...
+      <else>
+        <text variable="edition" suffix="."/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="locators">
+    <choose>
+      <if type="article-journal article-magazine" match="any">
+        <group delimiter=": " prefix=" ">
+          <group>
```
Comment (adam3smith) [inline behaviour.csl]: remove group (doesn't do anything)

File: behaviour.csl (hunk 1 above)
Comment (damnation333) [inline behaviour.csl]: Thanks. good spot. Double checked that with some actual papers and fixed as necessary.

**Comment by adam3smith** [comment]
Thanks!

**Pull request merged**